src/frontends/pytorch/src/op_table.cpp

```diff
@@ -341,6 +342,8 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::asinh_", op::inplace_op<op::translate_1to1_match_1_inputs<opset10::Asinh>>},
         {"aten::atan", op::optional_out<op::translate_1to1_match_1_inputs_with_fp32_type_alignment<opset10::Atan>, 1>},
         {"aten::atan_", op::inplace_op<op::translate_1to1_match_1_inputs<opset10::Atan>>},
+        {"aten::atan2", op::translate_atan2_op},
```

Comment:
I experienced some build issues there, either you need to remove `_op` part from function name (also in translation) or add this `_op` in `OP_CONVERTER(translate_atan2);`. Generally, in PyTorch Frontend we don't use _op at the end so it may be better to stick to existing name convention.